CMakeLists.txt

```diff
@@ -16,6 +16,7 @@
 
 cmake_minimum_required(VERSION 3.0.0)
 project(WABT VERSION 1.0.13)
+include(GNUInstallDirs)
```

Comment:
Why is these needed?

Reply:
`CMAKE_INSTALL_LIBDIR` isn't set without by default without them which would fail the install, not sure where the default location to install them is without it.

Not sure about the compatibility between OS outside of Linux either.

Binaries installed to `bin/` seem to be done manually here and I can try adapting that over to fit with `lib/`?